.travis.yml

```diff
@@ -14,7 +14,7 @@ matrix:
     jdk: oraclejdk8
     env: UPLOAD=true UPLOAD_NATIVE=true
   - os: linux
-    jdk: openjdk7
+    jdk: openjdk11
```

Comment:
Will this change cause jme artifacts to get build and deployed using java 11 ? 
In this case they wont run on Android unless we use `--release 8 ` flag for `javac`  (https://stackoverflow.com/questions/43102787/what-is-the-release-flag-in-the-java-9-compiler)

Reply:
I believe the artifacts are attached to the java8 Travis ci task

Reply:
Perhaps @stephengold can give us some insight into this?

Reply:
The `jdk: openjdk7` entry in the matrix doesn't have `UPLOAD` set, so it doesn't deploy anything. I believe it's there simply to catch commits that would prevent the Engine from building in a JDK7 environment.

Reply:
This should mean that because the openjdk7 entry became the openjdk11, it should still only deploy the openjdk8 build.

Reply:
Yes and no, iirc that distinction was made because the Engine was built in JDK7 environment but something (lwjgl3? openCL? vr?) needed the JDK8, so we added a separate build for that. Maybe the build.gradle can tell us more about that.
Maybe in the meantime it has become like stephen has outlined though.

However regarding @Ali-RS comment: If we use --release 8, then it's like using JDK 8 again, so no benefit at all as the modules won't be fancy-11-ish. That would mean we only have that to be future proof and to switch to 11 in a few years (when android supports it)